project/Dependencies.scala

```diff
@@ -31,7 +31,7 @@ object Dependencies {
   private val scalaParallelCollections =
     "org.scala-lang.modules" %% "scala-parallel-collections" % versions.scalaParallelCollections
 
-  private val testSeq = Seq(scalaTest, pekkoTestKit)
+  private val testSeq = Seq(scalaTest, pekkoTestKit, pekkoStream)
```

Comment:
This is needed otherwise you get

```
[info] org.zalando.kanadi.CommitCursorOmittedSpec *** ABORTED ***
[info]   java.lang.IllegalStateException: You are using version 1.0.2 of Apache Pekko, but it appears you (perhaps indirectly) also depend on older versions of related artifacts. You can solve this by adding an explicit dependency on version 1.0.2 of the [pekko-protobuf-v3, pekko-stream] artifacts to your project. Here's a complete collection of detected artifacts: (1.0.1, [pekko-protobuf-v3, pekko-stream]), (1.0.2, [pekko-actor, pekko-testkit]). See also: https://pekko.apache.org/docs/pekko/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed
```